src/cursor-doc/token-cursor.ts

```diff
@@ -676,7 +676,7 @@ export class LispTokenCursor extends TokenCursor {
     withinComment() {
         const cursor = this.clone();
         const token_type = cursor.getToken().type;
-        return token_type === 'comment' || (token_type === 'eol' && cursor.getPrevToken().type === 'comment');
+        return token_type === 'comment';
```

Comment:
Is it that either the token or the previous token is comment maybe? So both `|a|` and `|b|` here would be inComment:

```
|a|;; foo|b|<nl>
```

Reply:
My mistake - misinterpretation of what we had decided.